keep/alert/alert.py

```diff
@@ -82,6 +81,7 @@ def run_actions(self):
 
     def run(self):
         self.logger.debug(f"Running alert {self.alert_id}")
+        # todo: check why is this needed?
```

Comment:
?

Reply:
You've added the line
`        self.context_manager.set_alert_context(self._get_alert_context())`
but I couldn't see why it is needed